src/views/LoginForm/loginFormConstants.js

```diff
@@ -0,0 +1,12 @@
+export const loginFormInputs = [
```

Comment:
If this constant file will only contain static data. I'd suggest making it `.json` file